app/views/articles/_form.html.erb

```diff
@@ -13,6 +13,9 @@
           <%= f.label :description, class: "col-2 col-form-label text-light" %>
             <div class="col-10">
               <%= f.text_area :description, rows: 8, class: "form-control shadow rounded", placeholder: "Description of advertisement" %>
+                <label for="date">Expired at: </label>
+                <input type="date" id="date" name="date"/>
```

Comment:
1. Use f. to refer to real object
2. Use type="datetime-local"
3. Add existing value
4. Move it out of desctiption scope (OPTIONAL)